packages/scripts/src/new.ts

```diff
@@ -37,6 +43,79 @@ function main() {
   // changing README to use the adapter name instead of example
   const n_cap: string = n[0].toUpperCase() + n.slice(1)
   shell.sed('-i', 'Example', n_cap, `packages/${type}s/${n}/README.md`)
+
+  // pull latest workspace data after files have been generated
+  const current_workspace: WorkspacePackages = getWorkspacePackages() //using this alphabetizes everything
+  const adapter_list = current_workspace.filter(w => w.type === `${type}s`)
+  const adapter_names = adapter_list.map(w =>
+    w.descopedName
+      .split('-')
+      .slice(0, -1)
+      .join('-'),
+  )
+
+  // add to yarn workflow
+  const github_path = '.github/strategy/adapters.json'
+  const github_adapters = readJson(github_path)
+  github_adapters[`${type}s`].adapter = adapter_names // only replace the target adapter type
+  writeJson(github_path, github_adapters)
+
+  // add to packages/tsconfig.json
+  const tsconfig_path = 'packages/tsconfig.json'
```

Comment:
@HenryNguyen5 I tried using that, but typescript doesn't like that the file is outside of the `rootDir` and throws this error:
```
packages/scripts/src/new.ts:6:25 - error TS6059: File '/home/chainlink/Documents/external-adapters-js/packages/tsconfig.json' is not under 'rootDir' '/home/chainlink/Documents/external-adapters-js/packages/scripts/src'. 'rootDir' is expected to contain all source files.

6 import * as tsconf from '../../tsconfig.json'
                          ~~~~~~~~~~~~~~~~~~~~~

packages/scripts/src/new.ts:6:25 - error TS6307: File '/home/chainlink/Documents/external-adapters-js/packages/tsconfig.json' is not listed within the file list of project '/home/chainlink/Documents/external-adapters-js/packages/scripts/tsconfig.json'. Projects must list all files or use an 'include' pattern.

6 import * as tsconf from '../../tsconfig.json'
                          ~~~~~~~~~~~~~~~~~~~~~
```
So that's why I was using the `readJson` function that reads it via shelljs. Do I need to adjust something in the config files?

Reply:
You should be able to set the `rootDir` to the root of the repo for the scripts project. It makes sense to anyway since the scripts operate over the scope of the entire project.

Reply:
oooh that makes more sense. I saw that as a fix, but wasn't sure what other things might happen if I did